CMakeLists.txt

```diff
@@ -141,6 +141,7 @@ option(FLB_OUT_PLOT           "Enable Plot output plugin"           Yes)
 option(FLB_OUT_FILE           "Enable file output plugin"           Yes)
 option(FLB_OUT_TD             "Enable Treasure Data output plugin"  Yes)
 option(FLB_OUT_RETRY          "Enable Retry test output plugin"      No)
+option(FLB_OUT_PGSQL          "Enable PostgreSQL output plugin"      No)
```

Comment:
Curious: Why disable by default? (Not suggesting that's wrong, just want to understand).

Reply:
Yes because of out_kafka